Sources/TuistKit/Services/GraphService.swift

```diff
@@ -67,8 +84,14 @@ final class GraphService {
             try exportDOTRepresentation(from: graph, at: filePath)
         case .png:
             try exportPNGRepresentation(from: graph, at: filePath, layoutAlgorithm: layoutAlgorithm)
+        default:
+            throw GraphServiceError.withMessage("\(format.rawValue) is not a visual graph format to export")
         }
     }
+    
+    private func export(jsonContent: String, at filePath: AbsolutePath) throws {
+        try FileHandler.shared.write(jsonContent, path: filePath, atomically: true)
```

Comment:
Because the method is simple and it's only used called from a place, I recommend inlining the code here in the `run` method.

Reply:
Will do!